conanfile.py

```diff
@@ -68,11 +95,25 @@ def build(self):
             replace_in_file("%s/CMakeLists.txt" % self.ZIP_FOLDER_NAME, "project(libjpeg-turbo C)", "")
             
             cmake = CMake(self.settings)
-            self.run("cd %s && mkdir _build" % self.ZIP_FOLDER_NAME)
-            cd_build = "cd %s/_build" % self.ZIP_FOLDER_NAME
-            self.run('%s && cmake .. %s -DWITH_CRT_DLL=ON' % (cd_build, cmake.command_line)) # Don't change runtime, conan will take care of
-            self.run("%s && cmake --build . %s" % (cd_build, cmake.build_config))
-                
+            builddir = os.path.join(self.ZIP_FOLDER_NAME, "_build")
+
+            if os.path.exists(builddir):
```

Comment:
I reproduced the other error:

> SDL2_image/2.0.1@lasote/testing: Copying sources to build folder
> Traceback (most recent call last):
>   File "<string>", line 10, in <module>
>   File "<string>", line 6, in run
>   File "conan\conans\client\command.py", line 840, in main
>   File "conan\conans\client\command.py", line 739, in run
>   File "conan\conans\client\command.py", line 281, in test_package
>   File "conan\conans\client\manager.py", line 298, in install
>   File "conan\conans\client\installer.py", line 59, in install
>   File "conan\conans\client\installer.py", line 139, in _build
>   File "conan\conans\client\installer.py", line 189, in _build_node
>   File "conan\conans\client\installer.py", line 272, in _build_package
>   File "shutil.py", line 177, in copytree
>   File "os.py", line 157, in makedirs
> WindowsError: [Error 183] Eine Datei kann nicht erstellt werden, wenn sie bereits vorhanden ist: 'C:\\Users\\madoe\\.conan\\data\\SDL2_image\\2.0.1\\lasote\\testing\\build\\7d7911fc15eb9bcc07c73c17d54d0e3ad343607b'
> conan returned -1

Thats the error (appearing for SDL2_image here, but its been the same for libjpeg)
Translated the error is:

> WindowsError: [Error 183] Cannot create a file when that file already exists

The failure appears when conan is invoked with "--build"